docs/http-api/entry-api/delete_data.mdx

```diff
@@ -0,0 +1,324 @@
+                details: {
+                    name: ":entry_name",
+                    description: "Name of entry",
+                    isRequired: true,
+                },
+            },
+        ]
+    }
+    responses={
+        [
+            {
+                status: "200",
+                message: "OK",
+                summary: "The records have been removed",
+            }, {
+            status: "400",
+            message:
+                "Bad Request",
+            summary:
+                "Posted content bigger or smaller than content-length",
+        },
+            {
+                status: "401",
+                message:
+                    "Unauthorized",
+                summary:
+                    "Access token is invalid or empty",
+            },
+            {
+                status: "403",
+                message:
+                    "Forbidden",
+                summary:
+                    "Access token doesn't have the necessary permissions",
+            },
+            {
+                status: "404",
+                message:
+                    "Not Found",
+                summary:
+                    "The bucket or entry doesn't exist",
+            },
+            {
+                status: "422",
+                message:
+                    "Unprocessable Entity",
+                summary:
+                    "Malformed request",
+            },
+        ]}
+
+/>
+
+<SwaggerComponent
+    method="DELETE"
+    path="/api/v1/b/:bucket_name/:entry_name/q"
+    summary="Remove records within a time range"
+    description={
+        <>
+            The method removes records within a time range from an entry. The time range is specified in the query
+            parameters.
+            The method cam return errors for specific records that were not removed in the response headers according to
+            the Batch Protocol.
+
+            <br/>
+            Additionally, you can specify filters to remove only records that match the filter criteria.
+
+            <br/>
+            <br/>
+            If authentication is enabled, the method needs a valid API token with read access to the entry's bucket.
+            <br/>
+            <br/>
+            <strong> Changes: </strong>
+            <ul>
+                <li>
+                    Since 1.12, the method was introduced.
+                </li>
+            </ul>
+        </>
+    }
+    parameters={[
+        {
+            type: "path",
+            details: {
+                name: ":bucket_name",
+                description: "Name of bucket",
+                isRequired: true,
+            },
+        },
+        {
+            type: "query",
+            details: {
+                name: "start",
+                description:
+                    "A UNIX timestamp in microseconds. If not set, the query starts from the oldest record in the entry.",
+                dataType: "Integer",
+                isRequired: false,
```

Comment:
It's not mandatory to set the time range?

Reply:
No, the idea is that you can use labels over the entire dataset. However, I can add a check to return an error when no parameter is set to prevent deleting the entire entry.